domains/datagarden.localplayer.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "localplayer.dev",
+    "subdomain": "datagarden",
+
+    "owner": {
+        "email": "av.yigitdincsoy@gmail.com"
+    },
+
+    "record": {
+        "TXT": [""]
```

Comment:
TXT record with no data provided. TXT record alone isn't enough as it doesn't do anything.